bin/init.ml

```diff
@@ -249,4 +249,4 @@ let term =
     init @@ Test { context; common; options = () });
   print_completion kind name
 
-let command = (term, info)
+let command = Cmd.v info term
```

Comment:
This file (and probably others) mention "when cmdliner supports group, update it". we should create corresponding issues

Reply:
Done in #6161